config/initializers/extends.rb

```diff
@@ -35,7 +35,8 @@ class Extends
                            result_comments: 12,
                            project_activity: 13, # TODO
                            project_samples: 14, # TODO
-                           experiment: 15 }
+                           experiment: 15,
+                           my_module_repository: 16 }
```

Comment:
Thanks, nice catch! Damn, I hate that this is implemented this way... We should make it a "string" enum somehow, this is gonna cause us problems in the long run.